pkgs/development/python-modules/aw-client/default.nix

```diff
@@ -56,6 +57,10 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "aw_client" ];
 
+  passthru.updateScript = gitUpdater {
```

Comment:
you can just use nix-update

Reply:
Is it generally better to use nix-update if both nix-update and gitUpdater work? Do I need to set updateScript if either work?

Reply:
@SuperSandro2000 sorry to ping you since I know you're busy, but could you help point me in the right direction?

Reply:
It does not really matter tbh, I just always use ``nix-update name`` because it always works.